SpeechResponder/SpeechResponder.cs

```diff
@@ -34,6 +34,8 @@ public class SpeechResponder : EDDIResponder
         protected static List<Event> eventQueue = new List<Event>();
         private static readonly object queueLock = new object();
 
+        private readonly CottleHighlightingDefinition cottleHighlightingDefinition = new CottleHighlightingDefinition();
```

Comment:
This appears to be unused. Dead code?

Reply:
Instantiating this object installs the Cottle grammar definition into AvalonEdit. I'll also be using it to load and save any customisation by the user.

Reply:
Ok. My code analysis identified it as dead code so it might be worth dropping in a comment stating why it's needed.